website/_Layout.cshtml

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <meta name="description" content="Your one-stop-shop for high performance .NET graphics &amp; compute. A .NET Foundation project.">
+  @{
+      var title = Model.GetString("Title");
+  }
+  @if (title is null)
+  {
+      <title>
+        Silk.NET - High-Speed &amp; Advanced .NET Graphics &amp; Compute
+      </title>
+  }
+  else
+  {
+      <title>
+        @title - Silk.NET
+      </title>
+  }
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" rel="stylesheet" />
+  <link href="@InputUrl("theme/nucleo.css")" rel="stylesheet" />
+  <link href="@InputUrl("theme/fontawesome.css")" rel="stylesheet" />
+  <link href="@InputUrl("theme/argon.css")" rel="stylesheet" />
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/docsearch.js/2/docsearch.min.css" />
+  <link rel="stylesheet" href="@InputUrl("theme/docs.css")" />
+  <link rel="stylesheet" href="@InputUrl("theme/silk.css")" />
+</head>
+
+<body>
+  @Html.Partial("_theme/src/10-nav.cshtml")
+  @Html.Partial("_theme/src/20-contentroot.cshtml", new ViewDataDictionary(ViewData) { { "body", RenderBody() }})
+  <script src="@InputUrl("theme/jquery.min.js")" type="text/javascript"></script>
```

Comment:
These are a lot of JavaScript files, do they have to all be there?

Reply:
Deleted some which were pulled in with the template used that didn't look to be doing much.